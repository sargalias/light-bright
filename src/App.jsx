import React from 'react';
import LightBrightPage from 'pages/LightBrightPage';
import './base/reset.scss';
import './abstracts/_variables.scss';

const App = () => <LightBrightPage />;

export default App;
