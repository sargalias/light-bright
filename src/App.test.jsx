import React from 'react';
import { render } from 'testUtils'; // eslint-disable-line
import App from './App';

test('test', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
