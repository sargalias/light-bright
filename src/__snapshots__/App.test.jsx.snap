// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test 1`] = `
<div>
  <main
    class=""
  >
    <header
      class="_mainContentHeader"
    >
      <h1>
        Light Bright
      </h1>
    </header>
    <div
      class="_lightBrightGame"
    >
      <div
        class="_controls"
      >
        <button
          type="button"
        >
          Reset last color
        </button>
        <button
          type="button"
        >
          Reset all
        </button>
      </div>
      <div
        class=""
      >
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
        <div
          class=""
        />
      </div>
    </div>
    <section
      class="_instructions"
    >
      <h2
        class="_heading"
      >
        Instructions
      </h2>
      <ul>
        <li>
          <strong>
            Click:
          </strong>
           Paint a circle with a unique color.
        </li>
        <li>
          <strong>
            Click on a circle, then drag:
          </strong>
           Generate a unique color and paint circles with it.
        </li>
        <li>
          <strong>
            Reset last color:
          </strong>
           Unpaint the most recently painted circles.
        </li>
        <li>
          <strong>
            Reset all:
          </strong>
           Unpaint all circles.
        </li>
      </ul>
    </section>
    <div
      class=""
    >
      <p
        class="_text"
      >
        Â© Spyros Argalias
        2019
      </p>
      <p
        class="_text"
      >
        <a
          href="https://github.com/sargalias/light-bright"
          rel="noopener noreferrer"
          target="_blank"
        >
          View code
        </a>
      </p>
    </div>
  </main>
</div>
`;
